imagersite/imager_images/templates/photo.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+{% load thumbnail %}
+{% block content %}
+    <link rel="stylesheet" type='text/css' href="//cdnjs.cloudflare.com/ajax/libs/sweetalert/1.1.0/sweetalert.min.css"/>
+    <script type="text/javascript" src="//code.jquery.com/jquery-2.1.4.min.js"></script>
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/d3/3.5.6/d3.min.js"></script>
+    <script type="text/javascript" src="//algorithmia.com/v1/clients/js/algorithmia-0.2.0.js"></script>
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/sweetalert/1.0.1/sweetalert-dev.js"></script>
+    <script src="http://d3js.org/d3.v3.min.js" charset="utf-8"></script>
+
+    <div id="ww">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 centered">
+                    <main id="body">
+                        <div><h2>{{ user }}</h2>
+                            {% if user.is_authenticated %}
+
+                            <h3>{{ photo.title}}</h3>
+
+                            <div class='grid'>
+                            <div id='photo-container'>
+                                {% thumbnail photo.image "600x600" as image %}
+                                <img src="{{ image.url }}">
+                                {% endthumbnail %}
+                            </div>
+                            <!-- <p>No photo...</p>
+                            <img src='/static/img/is.jpg' height='180' width='180'> -->
+                            <div class='info'>
+                            <p>{{ photo.description }}</p>
+
+                            <div>
+
+                            {% if not faces %}
+                              <a href="{% url 'detect_faces' object.id %}"><button>Detect Faces</button></a>
+                            {% endif %}
+                              <script type="text/javascript">
+                            FaceTagger.init({
+                              container: '#photo-container',
+                              labelUrl: '{% url "edit_face" object.id %}',
+                              csrf: '{{ csrf_token }}',
+                              originalWidth: {{ object.image.width }},
+                              originalHeight: {{ object.image.height }},
```

Comment:
since you use the name `photo` to refer to the photo object used to render this page in other locations (see L23 above), why change that pattern here?  it's a bit confusing to have you using `photo` in one place and `object` in another to refer to the exact same item.